sdk/identity/azure-identity/samples/CMakeLists.txt

```diff
@@ -31,22 +31,22 @@ target_link_libraries(workload_identity_credential_sample PRIVATE azure-identity
 target_include_directories(workload_identity_credential_sample PRIVATE .)
 create_per_service_target_build_for_sample(identity workload_identity_credential_sample)
 
-add_executable(client_secret_credential_sample client_secret_credential.cpp)
-target_link_libraries(client_secret_credential_sample PRIVATE azure-identity service get-env-helper)
-target_include_directories(client_secret_credential_sample PRIVATE .)
-create_per_service_target_build_for_sample(identity client_secret_credential_sample)
-
-add_executable(default_azure_credential_sample default_azure_credential.cpp)
-target_link_libraries(default_azure_credential_sample PRIVATE azure-identity service)
-target_include_directories(default_azure_credential_sample PRIVATE .)
-create_per_service_target_build_for_sample(identity default_azure_credential_sample)
-
-add_executable(environment_credential_sample environment_credential.cpp)
-target_link_libraries(environment_credential_sample PRIVATE azure-identity service)
-target_include_directories(environment_credential_sample PRIVATE .)
-create_per_service_target_build_for_sample(identity environment_credential_sample)
-
-add_executable(managed_identity_credential_sample managed_identity_credential.cpp)
-target_link_libraries(managed_identity_credential_sample PRIVATE azure-identity service)
-target_include_directories(managed_identity_credential_sample PRIVATE .)
-create_per_service_target_build_for_sample(identity managed_identity_credential_sample)
+#add_executable(client_secret_credential_sample client_secret_credential.cpp)
+#target_link_libraries(client_secret_credential_sample PRIVATE azure-identity service get-env-helper)
+#target_include_directories(client_secret_credential_sample PRIVATE .)
+#create_per_service_target_build_for_sample(identity client_secret_credential_sample)
+
+#add_executable(default_azure_credential_sample default_azure_credential.cpp)
+#target_link_libraries(default_azure_credential_sample PRIVATE azure-identity service)
+#target_include_directories(default_azure_credential_sample PRIVATE .)
+#create_per_service_target_build_for_sample(identity default_azure_credential_sample)
```

Comment:
Please explain why we are commenting all the samples out.

Reply:
I am not sure I understand the issue.

It's reasonable for us to disable building/running the samples if we know the problem and have a plan to update it (even in a separate, follow-up PR).

Can you share the error you were seeing or the link to the failing pipeline that required us to comment these out?

Reply:
Looking at the pipelines that build and run Identity samples, they seem to pass from this PR, without commenting them out:
https://github.com/Azure/azure-sdk-for-cpp/pull/5785

https://dev.azure.com/azure-sdk/internal/_build/results?buildId=3955376&view=logs&j=ea109acd-b7d3-5ded-4f2b-e0ac8b0c5651&t=88917f58-d2f9-5f14-5936-80f13f049405

https://dev.azure.com/azure-sdk/internal/_build/results?buildId=3955376&view=logs&j=fad7e5d9-ee9d-5455-29cf-12d77fc3497f&t=e819b337-613f-5b6a-fedf-c6b8af28994f

Reply:
resolving this conversation to merge the CI